src/_guides/language/language-tour.md

```diff
@@ -441,7 +452,8 @@ const msUntilRetry = secondsUntilRetry * msPerSecond;
 
 ### Strings
 
-A Dart string is a sequence of UTF-16 code units. You can use either
+A Dart string is a sequence of UTF-16 code units. (When Dart is compiled to
+JavaScript, the string representation remains the same.) You can use either
```

Comment:
Not sure what the addition gives us here. 
It's basically saying "A is true. Even when B, A is still true." If I didn't have the expectation that B would change anything, now I'm just confused :(

Is this because we have previously said that the same isn't true for `int`?
Otherwise, I'd just drop the sentence, or change it to something like:
> , the same as a JavaScript, Java or C# string.
(except that doesn't flow very well, so maybe something like that as written by a professional writer)

Reply:
I think I'll just drop it. In other places we talk about JavaScript comparisons, but we should probably move away from those.